kibana/templates/deployment.yaml

```diff
@@ -76,11 +85,36 @@ spec:
       imagePullSecrets:
 {{ toYaml .Values.imagePullSecrets | indent 8 }}
     {{- end }}
+      initContainers:
+      - name: configure-kibana-token
```

Comment:
@elastic/kibana-security can you review this approach?  short version:  interactive setup isn't an option in a helm environment, so we're manually configuring kibana similar to the cli.

- a kibana service account token is created
- certificates are added
- hostname is set